1-exercises/B-setting-values/exercise2.js

```diff
@@ -27,6 +29,10 @@ let student = {
 
 // write code here
 
+if (student["attendance"] >= 90 && student["examScore"]> 60){
+  student["hasPassed"] = true;
+}
+
```

Comment:
well done hengameh :), I am not sure if you needed to show false value too, but you can add else { student["hasPassed"] = false;}